crates/sui/tests/protocol_version_tests.rs

```diff
@@ -11,8 +11,11 @@ use test_utils::authority::start_node;
 async fn test_validator_panics_on_unsupported_protocol_version() {
     let dir = tempfile::TempDir::new().unwrap();
     let network_config = sui_config::builder::ConfigBuilder::new(&dir)
-        .with_protocol_version(ProtocolVersion::new(2))
-        .with_supported_protocol_versions(SupportedProtocolVersions::new_for_testing(1, 1))
+        .with_protocol_version(ProtocolVersion::new(ProtocolVersion::MAX.as_u64() + 1))
```

Comment:
you can do:

```
with_protocol_version(ProtocolVersion::MAX + 1)
```

Reply:
I am going to do that on main where it matters...